tests/test_reader.py

```diff
@@ -59,7 +59,7 @@ def createFile(filename):
 def deleteFile(filename):
     os.remove(filename)
 
-def test():
+def test_main():
```

Comment:
Shouldn't this be something more descriptive, like `test_sudoko_reader` for example.

Reply:
You're right, Mo! I also noticed that there were many different spellings of sudoku in this file, I fixed those typos :)